includes/class-wc-payments-payment-method-messaging-element.php

```diff
@@ -48,16 +48,16 @@ public function __construct( WC_Payments_Account $account, $gateway ) {
 	 */
 	public function init(): string {
 		global $product;
-		$currency_code   = get_woocommerce_currency();
-		$store_country   = WC()->countries->get_base_country();
-		$billing_country = WC()->customer->get_billing_country();
-
+		$currency_code      = get_woocommerce_currency();
+		$store_country      = WC()->countries->get_base_country();
+		$billing_country    = WC()->customer->get_billing_country();
 		$product_variations = [
 			'base_product' => [
 				'amount'   => WC_Payments_Utils::prepare_amount( $product->get_price(), $currency_code ),
 				'currency' => $currency_code,
 			],
 		];
+
```

Comment:
Re-grouping `product_variations` in this case is not making sense, it logically belongs to foreach-construct.

Reply:
That makes sense. I have regrouped this with the foreach-construct since they are logically connected. 8275876